### ISSUE TITLE
Sound fluctuates after turning off Voicing on iPad

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/quadrilateral#423]
**Test device**
iPad 9th generation

**Operating System**
iOS 16.3.1

**Browser**
safari

**Problem description**
For https://github.com/phetsims/qa/issues/923, after turning off Voicing the Shape Sounds will fade in and out. 

**Steps to reproduce**
1. Move a side/corner to begin sound 
2. Before sound fades, turn on Play Shape Sounds Forever
3. Turn on Voicing
4. Turn off Voicing and exit Preferences

**Visuals**
Here is what is sounds like after the steps above:

https://user-images.githubusercontent.com/87318828/228543891-0e1d0304-6e03-42b9-9a85-6d0ae987e569.mov

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Quadrilateral‬
URL: https://phet-dev.colorado.edu/html/quadrilateral/1.0.0-dev.76/phet/quadrilateral_all_phet.html
Version: 1.0.0-dev.76 2023-03-28 17:13:45 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Safari/605.1.15
Language: en-US
Window: 1342x705
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi - Just in case, I tried on Win 10 Chrome but did not see it there so Ill try on iOS. This is probably a common code problem with audio ducking + speech synthesis (https://github.com/phetsims/tambo/issues/167)

**Comment by jessegreenberg** [comment]
I am able to reproduce this with iOS 15.0.2 Safari. I could not reproduce this in macOS 11.7.2 Safari.

**Comment by jessegreenberg** [comment]
- My first thought was that the synth was trying to speak with empty strings, causing the soundManager to duck/un-duck over and over. But I put a log statement here: https://github.com/phetsims/tambo/blob/4e4aab289995d7f6e3ddb0f77116618a6aa33b43/js/soundManager.ts#L257-L267 and I do not see `updateDuckingState` getting called while the sounds fluctuate.

**Comment by jessegreenberg** [comment]
I found it. It is caused by this workaround that was added for Chromebooks: https://github.com/phetsims/utterance-queue/blob/36fdea460883369af36712e1aefef89bf2e25e0d/js/SpeechSynthesisAnnouncer.ts#L472-L483

What must be happening is that iOS has built-in audio ducking for speech synthesis. That is why removing PhET's audio ducking didn't change the behavior. The empty string in that workaround still triggers the ducking even though there is no speech.

I am guessing this was also introduced when the workaround string became an empty space for https://github.com/phetsims/friction/issues/328.
